### PR TITLE
chore(merge): release->edge for 4.0

### PR DESCRIPTION
**Pull request by sfoster1**
This merges release into edge with changes from 4.0.0, and is a working branch so tests can run and we can check the merge status.

### CODE REVIEW

**Review by Laura-Danielle — APPROVED** [review]
looks good to me

File: api/src/opentrons/api/session.py [hunk 1]
```diff
@@ -765,28 +697,20 @@ def _get_labware(command):  # noqa(C901)
...
-            # named tuple like location descends from tuple and therefore
-            # passes the check
-            containers.append(get_container(location))
-        elif isinstance(location, Location):
+        if isinstance(location, Location):
             if location.labware.is_well:
                 containers.append(location.labware.parent.as_labware())
             elif location.labware.is_labware:
                 containers.append(location.labware.as_labware())
+        elif isinstance(location, (labware.Well, labware.Labware)):
```
Comment (amitlissack) [inline api/src/opentrons/api/session.py:705]: Very good!

File: robot-server/robot_server/service/legacy/routers/deck_calibration.py [hunk 2]
```diff
@@ -1,107 +1,34 @@
-from uuid import UUID
+from fastapi import APIRouter, Depends
```
Comment (amitlissack) [inline robot-server/robot_server/service/legacy/routers/deck_calibration.py:1]: We should change the name of this file at some point

**Review by amitlissack — APPROVED** [review]
HMG deserves a big hug and cake for this. Well done!

**Pull request merged**